_includes/sub-template-create-update-wls-on-aks.md

```diff
@@ -0,0 +1,143 @@
+## Prepare the Parameters
+
+You must construct a parameters JSON file containing the parameters to be passed to the ARM template.
+For background information about parameter files, see [Create Resource Manager parameter file](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/parameter-files). 
+This section shows how to obtain the values for the required properties.
+
+### Obtain parameter values from Azure portal
+
+The following steps use the [Azure Create UI Definition Sandbox](https://portal.azure.com/?feature.customPortal=false#blade/Microsoft_Azure_CreateUIDef/SandboxBlade) to obtain the values. If you prefer to edit a json file, you can also create the objects in your parameters file directly. The Azure UI Definition Sandbox provides controls to select resources and input your values easily.  **More importantly, this approach generates syntactically valid JSON, eliminating an important class of data entry error.**
```

Comment:
json file -> JSON file

Reply:
Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb